example/android/settings.gradle

```diff
@@ -1,6 +1,6 @@
 rootProject.name = 'example'
 
 apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle");
-applyNativeModulesSettingsGradle(settings, "../..")
+applyNativeModulesSettingsGradle(settings);
```

Comment:
why this change? does autolinking on android still work?

Reply:
yes, the second argument is deprecated on rn 62 applying the `"@react-native-community/datetimepicker": "./"` in package.json the autolink works successfull